docs/resources/Audit_Log.md

```diff
@@ -99,6 +99,9 @@ Whenever an admin action is performed on the API, an entry is added to the respe
 | THREAD_UPDATE                         | 111   |
 | THREAD_DELETE                         | 112   |
 | APPLICATION_COMMAND_PERMISSION_UPDATE | 121   |
+| AUTO_MODERATION_RULE_CREATE           | 140   |
+| AUTO_MODERATION_RULE_UPADTE           | 141   |
```

Comment:
Update?

Reply:
changing the config of a rule, adding removing a keyword, changing log channel ect

Reply:
i think they are talking about the typo

Reply:
oh 💀